pkg/comp-functions/functions/common/interfaces.go

```diff
@@ -39,3 +39,15 @@ type AllowedNamespaceGetter interface {
 	GetAllowAllNamespaces() bool
 	GetAllowedNamespaces() []string
 }
+
+// Required to get info required for alerting.
+type Alerter interface {
+	GetVSHNMonitoring() vshnv1.VSHNMonitoring
+	GetInstanceNamespace() string
+}
+
+// minimalist PDB interface
+type PDB interface {
```

Comment:
Both of these functions are already part of the `InfoGetter` interface.

If you want to keep them separate, you could remove those two functions from the `InfoGetter` interface and then add the `PDB` interface to `InfoGetter`. This way we'll have the definition exactly once.

Reply:
I prefer it that way as I added one new function that is usable only by PDB

Reply:
With my suggestion it would still be the same, we'd just get rid of the duplication.

Reply:
not, because InfoGetters started to complain that they do not implement specific method:
```
cannot use comp (variable of type *"github.com/vshn/appcat/v4/apis/vshn/v1".VSHNNextcloud) as common.Composite value in argument to common.BootstrapInstanceNs: *"github.com/vshn/appcat/v4/apis/vshn/v1".VSHNNextcloud does not implement common.Composite (missing method GetPDBLabels)
```

Reply:
with such separation it will be easier for us to catch issues imho as compiler tells You there's a problem